contracts/BridgeAsset/BridgeAsset.go

```diff
@@ -1,3 +1,6 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
 // Code generated - DO NOT EDIT.
 // This file is a generated binding and any manual changes will be lost.
```

Comment:
I think these should be left out, they are auto-generated files, the license doesn't really apply, it will get removed anytime we rebuild them.